### PR TITLE
admin can view, edit, delete user task

### PR DESCRIPTION
**Pull request by NgocVu1994**

### CODE REVIEW

File: app/models/task.rb [hunk 1]
```diff
@@ -29,22 +29,22 @@ def assign_trainees_to_task
...
+  def change_user_task old_assigned_trainee
+    user_task = UserTask.find_by user: old_assigned_trainee, task: self
     if user_task.nil?
       self.user_tasks.create user: assigned_trainee,
         user_subject: user_subject
     else
       user_task.update_attributes user: assigned_trainee,
         user_subject: user_subject
-    end
+     end
```
Comment (thiennv57) [inline app/models/task.rb]: indent

**Comment by truonglocbinh** [comment]
ok

**Comment by thiennv57** [comment]
OK

File: app/controllers/admin/tasks_controller.rb [hunk 2]
```diff
@@ -32,14 +35,28 @@ def create
...
+    unless params[:task][:assigned_trainee_id].nil?
+      @task.change_user_task @old_assigned_trainee
+    end
     redirect_to :back
   end
 
+  def destroy
+    if @task.destroy
+      flash[:success] = flash_message "deleted"
+      redirect_to admin_course_subject_path(@course_subject.course, @course_subject.subject)
```
Comment (dieunb) [inline app/controllers/admin/tasks_controller.rb]: Dài quá 80 ký tự nhé

**Pull request merged**